api/origin_trials_api.py

```diff
@@ -119,6 +120,15 @@ def _validate_creation_args(
 
     return validation_errors
 
+  def check_post_permissions(self, feature_id) -> flask.Response | dict:
```

Comment:
So as long as users have a Google or Chromium account, they can edit the feature? By edit, it is the permission to edit the feature or e.g. request an extension?

Reply:
> So as long as users have a Google or Chromium account, they can edit the feature?

No, a couple of years ago it was the case that any @google.com or @chromium.org account could edit any issue, but now editing is limited to just users listed on the feature entry (and site admins and site editors).  This PR does not change the permission to edit any of the fields of the feature entry other than requesting an OT.

> By edit, it is the permission to edit the feature or e.g. request an extension?

In the source code "edit" permission refers to the ability to edit any of the fields of the feature entry.  Here, I am adding back the @google.com and @chromium.org rule just for requesting an OT.